index.html

```diff
@@ -56,8 +55,8 @@ <h2>I'm a Software Developer</h2>
             <h2>My Recent Works</h2>
             <div class="topLine"></div>
         </div>
-        <div class="grid">
-            <div class="mainCard">
+        <div class="grid" id="projectsSection">
+            <!-- <div class="mainCard">
                 <div class="cardImage">
                     <img src="Sources/MultiPost Stories.svg" alt="MultiPostImage">
                 </div>
```

Comment:
- [ ] kindly remove the commented piece of code for cleaner code practices. This is to follow the YAGNI  principle, which implies that you should only implement needed material in your code.